api/src/main/java/org/pmiops/workbench/api/WorkspacesController.java

```diff
@@ -192,6 +192,8 @@ public ResponseEntity<Workspace> createWorkspace(Workspace workspace) throws Bad
     }
     final Workspace createdWorkspace = workspaceMapper.toApiWorkspace(dbWorkspace, fcWorkspace);
     workspaceAuditor.fireCreateAction(createdWorkspace, dbWorkspace.getWorkspaceId());
+    if (cdrVersion.getTanagraEnabled()) {}
+
```

Comment:
Is this just a place holder? Maybe you should mark this-location in the ticket that will implement the logic here and remove these lines. 195-196 code does nothing

Reply:
Thanks, was testing if the flag was accessible and forgot to remove this line of code. Removed.